frontend/public/components/utils/link.tsx

```diff
@@ -33,6 +33,17 @@ export const namespacedPrefixes = [
   '/status',
 ];
 
+export const CLUSTER_SCOPED_ROUTES = [
```

Comment:
I worry about having a hard-coded list of paths that we need to maintain. It also means we won't be able to support routes contributed by plugins.